### PR TITLE
Add backup support

### PR DESCRIPTION
**Pull request by ruediste**
I need to backup my elasticsearch cluster. Therefore the path.repo needs to be specified. I also included some instructions.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -5,6 +5,9 @@ MAINTAINER pjpires@gmail.com
 # Override config, otherwise plug-in install will fail
 ADD config /elasticsearch/config
 
+# directory needs to be in place for startup
+RUN mkdir /backup
```
Comment (pires) [inline Dockerfile]: Can we pass `/backup` as an environment variable?

**Comment by ruediste** [comment]
Sorry for the wrong email. Fixed it. Also, see the code comment.

File: Dockerfile (hunk 1 above)
Comment (pires) [reply Dockerfile]: I think so.

**Comment by jjungnickel** [comment]
Can I help completing this MR? We'd like to use this :-)

**Comment by pires** [comment]
@jjungnickel be my guest for as long as you work with @ruediste.

**Comment by ruediste** [comment]
Hey, I'll look into it tomorrow. I'll come back to you then. Regards

**Comment by ruediste** [comment]
I added the `REPO_LOCATIONS` env var. 

@jjungnickel Could you test this?

**Comment by pires** [comment]
Unfortunately, this PR was never in a good shape and it won't be merged. I have however adapted one of the changes directly into the base container image for this repo, https://github.com/pires/docker-elasticsearch/blob/34045fe6f66528a10ecf137569f1f30bd1f70789/README.md#backup.

Thank you @ruediste!

**Pull request closed without merging**